public/app/plugins/datasource/tempo/QueryField.tsx

```diff
@@ -193,6 +193,8 @@ class TempoQueryFieldComponent extends React.PureComponent<Props, State> {
             query={query}
             onChange={onChange}
             onBlur={this.props.onBlur}
+            app={app}
+            onClearResults={this.onClearResults}
```

Comment:
I've actually been a fan of this for a while, but sometimes add newer props to the end especially if there are lots. I think it would be interesting to add some kind of linting for this but not overly pushed right now.